Stellar-transaction.x

```diff
@@ -1657,6 +1658,8 @@ case INVOKE_HOST_FUNCTION_SUCCESS:
 case INVOKE_HOST_FUNCTION_MALFORMED:
 case INVOKE_HOST_FUNCTION_TRAPPED:
     void;
+case INVOKE_HOST_FUNCTION_ERROR:
+    SCVal error;
```

Comment:
What is the ScVal in this case?

Reply:
The host and contracts use `Status` for errors, not SCVal.

Reply:
Yeah this should be a Status.

Reply:
```suggestion
case INVOKE_HOST_FUNCTION_ERROR:
    SCStatus error;
```

Reply:
I pushed a further commit that makes this change.

Reply:
Thanks, I was just not sure which type would be easier to consume here.